src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -40,8 +40,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         If(preprocessorSymbols Is Nothing, DefaultPreprocessorSymbols, ImmutableArray.CreateRange(preprocessorSymbols)),
                         ImmutableDictionary(Of String, String).Empty)
 
+            ' We test the mapped value, _languageVersion, rather than tha parameter, languageVersion,
```

Comment:
I have no idea what "Need to recheck wording" means.

---
In reply to: [72688595](https://github.com/dotnet/roslyn/pull/12796#discussion_r72688595) [](ancestors = 72688595)